doc/smart-switch/ip-address-assigment/smart-switch-ip-address-assignment.md

```diff
@@ -149,14 +175,8 @@ Based on the preset `t1-smartswitch` default topology the configuration generate
             "subtype": "SmartSwitch",
         }
     },
-    "MID_PLANE_BRIDGE" {
-        "GLOBAL": {
-            "bridge" : "bridge_midplane",
-            "address": "169.254.200.254/24"
-        }
-    },
     "DHCP_SERVER_IPV4": {
-        "bridge_midplane": {
+        "bridge-midplane": {
```

Comment:
The name of the bridge here doesn't match the name in `platform.json` midplane_network table:
"midplane_bridge" vs "bridge-midplane"

Reply:
Fixed.